editoast/src/views/timetable/stdcm.rs

```diff
@@ -251,7 +258,34 @@ async fn stdcm(
 
     let stdcm_response = stdcm_request.fetch(core_client.as_ref()).await?;
 
-    // 6. Handle STDCM Core Response
+    // 6. Check if the current tracing level is debug or lower, and if so, log STDCM request and response
+    if tracing::Level::DEBUG <= tracing::level_filters::LevelFilter::current() {
+        let stdcm_response_for_spawn = stdcm_response.clone();
+        let _ = tokio::spawn(
+            async move {
+                match &mut db_pool.get().await {
+                    Ok(conn) => {
+                        let trace_id = tracing::Span::current()
+                            .context()
+                            .span()
+                            .span_context()
+                            .trace_id();
+                        let _ = StdcmLog::changeset()
```

Comment:
```suggestion
                        StdcmLog::changeset()
```

this error shouldn't be ignored. We should log something if this fails.